src/api/modelTypes.js

```diff
@@ -100,6 +100,12 @@ export type User = {|
   bot_type?: number,
   bot_owner?: string,
 
+  // Tim mentions in 2020-02, at
```

Comment:
Small thing in this commit message: I've liked using the prefix "api types" for changes here. The "api" name is helpful for saying it's inside `src/api/`, the bindings to the Zulip server API. So in this case `api types [nfc]:`.

(At one level, all types changes are NFC because they don't affect what happens at runtime. But this is more so because it's only in the comments: it doesn't affect what future changes would type-check. So when it already says "types" I generally reserve "[nfc]" for changes, like this one, that make no functional changes even to the types.)